### DISCUSSION TITLE
Sequential commit

### DISCUSSION THREAD

**Discussion by zajck** [bosonprotocol/BPIPs#17] [BPIPs discussions]
Start of the discussion about #16.

**Comment by thecryptofruit** [comment]
LGTM 👍 I've suggested to add a note on "less-then or equal" secondary prices.

**Comment by thecryptofruit** [comment]
Just to note here that [BPIP-5](https://github.com/bosonprotocol/BPIPs/commits/main/content/BPIP-5.md) has recently been updated and added more details on Royalty Registry https://royaltyregistry.xyz/ etc...

**Comment by thecryptofruit** [comment]
Moving to Final as no further comments and RC tag ready.
